src/style.css

```diff
@@ -0,0 +1,61 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  margin-top: 20px;
+}
+
+.input {
+  box-sizing: border-box;
+  width: 100%;
+  border: 1px solid;
+  border-color: #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300  ;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  outline: none;
+  font-family: "Avenir-Heavy", sans-serif;
+}
+
+.biggie {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  margin-bottom: 20px;
+  line-height: 22px;
+}
+
+.bigSearch {
```

Comment:
Naming. I'd think of it as a search input at first glance - `bigSearchImage` would improve the predictability